### PR TITLE
Docs: Added overview of data concept in Dashboards.

### PR DESCRIPTION
**Pull request by bre1470**

### CODE REVIEW

File: docs/dashboards/data-handling.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+addition DataTable also emit events, when changes on the table happen.  Changes
+can be directly done on the table or with DataModifiers.
+
+
+
+## DataModifier
+
+DataTables can provide automatic modification of data during their lifetime with
+the help of modifiers.  Each table can have only one modifier, but this modifier
+might call additional modifiers.  If a DataModifierfor changes the DataTable,
```
Comment (pawellysy) [inline docs/dashboards/data-handling.md:41]:
```suggestion
might call additional modifiers.  If a DataModifier changes the DataTable,
```

File: docs/dashboards/data-handling.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+changes are available on a clone in the `DataTable.modified` property.
+
+
+
+## DataCursor
+
+The DataCursor provides a system to synchronize status data on cells of
+DataTables.  This can be for example highlights, markers, or selections.  It
+supports multiple tables and status simultaneously.  Each Board has a DataCursor
```
Comment (pawellysy) [inline docs/dashboards/data-handling.md:50]:
```suggestion
supports multiple tables and statuses simultaneously.  Each Board has a DataCursor
```

**Review by pawellysy — APPROVED** [review]
LGTM! Minor typos

File: docs/dashboards/data-handling.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+Highcharts Dashboards has a separate set of classes to handle data.  The
+following classes are used by the Board class to provide its components with
+data to show.
+
+
+
+## DataPool
+
+With the `BoardOptions.dataPool` option you can define as many connections to
```
Comment (karolkolodziej) [inline docs/dashboards/data-handling.md:12]: I would describe it as:
`... You can define as many connections to your data sources as you like.`

**Review by karolkolodziej — APPROVED** [review]
Maybe for each section, we might add a reference to a place where it is described in greater detail. What do you think?

**Pull request merged**